lib/crypto/tasks.py

```diff
@@ -24,43 +24,51 @@ def __exit__(self, type, value, traceback):
 
 
 @task
-def sign_addons(addon_ids, **kw):
+def sign_addons(addon_ids, force=False, **kw):
```

Comment:
I notice that hits iterates over all addon_ids. Should we consider spreading the load across the celery workers for large lists of addon ids? You could chunk the IDs and then sign those chunks and make this go faster. We wouldn't want to DOS the signing server however.